### PR TITLE
Updated shrinkwrap-resolver to 3.1.4 to fix download of wildfly-dist from http url

### PR DESCRIPTION
**Pull request by WolfgangHG**
Previous version 2.0.0-beta-3 caused on error on downloading wildfly-dist from maven, as it used a http url. Recent shrinkwrap-resolver uses a https url.

This pull request replaces https://github.com/arquillian/arquillian-extension-warp/pull/72 by dependabot.

### CODE REVIEW

File: build/resources/src/main/java/org/arquillian/warp/ftest/installation/ContainerInstaller.java [hunk 1]
```diff
@@ -118,6 +122,9 @@ public void uninstallContainer(@Observes UninstallContainer event) {
 
             if (containerHome.exists()) {
                 FileUtils.deleteQuietly(containerHome);
+                if (FileUtils.deleteQuietly(containerHome) == false) {
```
Comment (chengfang) [inline build/resources/src/main/java/org/arquillian/warp/ftest/installation/ContainerInstaller.java:125]: delete is called twice. Should remove line 124?

File: build/ftest-base/pom.xml [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
   <properties>
     <version.shrinkwrap>1.2.6</version.shrinkwrap>
-    <version.shrinkwrap.resolver>2.0.0-beta-3</version.shrinkwrap.resolver>
+    <version.shrinkwrap.resolver>3.1.4</version.shrinkwrap.resolver>
```
Comment (chengfang) [inline build/ftest-base/pom.xml:17]: we should reuse the same property from root pom, and remove the above property in this pom.

File: build/resources/src/main/java/org/arquillian/warp/ftest/installation/ContainerInstaller.java (hunk 1 above)
Comment (WolfgangHG) [reply build/resources/src/main/java/org/arquillian/warp/ftest/installation/ContainerInstaller.java:124]: D'oh, yes, copy/paste error. Thanks for spotting this. Fixed.

The WildFly 8.2.1 container does not seem to stop properly and thus will only be deleted partially. So I added the additional output, but I don't have an idea how to resolve it. See e.g. [https://github.com/ingwarsw/arquillian-suite-extension/issues/27](https://github.com/ingwarsw/arquillian-suite-extension/issues/27) for a similar problem.

File: build/ftest-base/pom.xml (hunk 2 above)
Comment (WolfgangHG) [reply build/ftest-base/pom.xml]: Though about this, too, but didn't dare to do it before. Is changed now.

**Review by chengfang — APPROVED** [review]

**Comment by chengfang** [comment]
Thanks @WolfgangHG !

**Pull request merged**